src/solvers/flattening/boolbv.h

```diff
@@ -104,6 +104,8 @@ class boolbvt:public arrayst
     return endianness_mapt{type, little_endian, ns};
   }
 
+  virtual endianness_mapt endianness_map(const typet &type) const;
```

Comment:
Should this be the only public interface for `endianness_map`?

Reply:
No, because byte extracts do (potentially) override the configured endianness.